exporter/stackdriverexporter/stackdriver.go

```diff
@@ -32,7 +33,10 @@ import (
 	"google.golang.org/grpc"
 )
 
-const name = "stackdriver"
+const (
+	name           = "stackdriver"
+	defaultTimeout = 12 * time.Second // Consistent with Cloud Monitoring's timeout
```

Comment:
Just want to make sure, 12s is the default deadline, not timeout. 
I don't know opencensus implementation of exporter but It's more important that deadline is properly set.

Reply:
Can you point me to the deadline parameter that needs to be set? Is it some gRPC client config?

The timeout I'm overriding here is https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/db101e30979316cca594a74b3181a9a3b6094086/stackdriver.go#L289
It's being used for `context.WithTimeout`, which internally calls `context.WithDeadline`: https://golang.org/pkg/context/#WithTimeout, so when it times out you will get an "deadline exceeded" error.